controllers/SiteController.php

```diff
@@ -308,26 +294,103 @@ public function actionChangeLanguage($lang)
         }
     }
 
-    public function actionDesignList()
-    {
-        return $this->render('design-list');
+    public function actionDesignList($viewMode = NULL) {
+        if (!empty($viewMode)) {
+            $viewMode = 1;
+        } else {
+            $viewMode = 0;
+        }
+        $query = new Query;
```

Comment:
@sapkaljagruti Instead of writing raw SQL query you may try `ActiveQuery`. [Reference.](https://www.yiiframework.com/doc/api/2.0/yii-db-activequery)